dataframe/dataframe.go

```diff
@@ -255,17 +257,20 @@ func (df DataFrame) Set(indexes series.Indexes, newvalues DataFrame) DataFrame {
 	if df.Err != nil {
 		return df
 	}
+	if indexes == nil {
+		return DataFrame{Err: createErr("nil indexes", "df.Set()")}
```

Comment:
Yes indeed, but that's what I was referring to with this line:

> There aren't multiple stacks/functions that push the error up before it is returned.

So the errors in `DataFrame.Err` are never going to be chained. And so I don't think they should be wrapped errors, but just a normal error.